### ISSUE TITLE
A11y: Improve keyboard interaction on carousel

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#4240]
## Bug Description

There are two issues with the page carousel:

1. Only the current page should be in the tab order - otherwise, if you have a 20-page story, you'll tab for a long to get from the canvas to the design panel, as you have to go through all the pages in the process.
2. The current page should have a clear focus indicator if focused - and this should be implemented in both collapsed carousel, regular carousel, and in the page grid view.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by dmmulroy** [comment]
This covers #3653

**Comment by csossi** [comment]
Tested as per instructions in 4248:
Testing Instructions
In story editor, tab through page and see that only active page indicator is focusable out of the page indicators, also that it's visually clear that the page indicator is focused.
In the story editor, click the grid view. In the grid view tab through the page and see that only the active page indicator is focusable out of the page indicators and that it's visually clear that the page indicator is focused.

Verified in QA
